### ISSUE TITLE
any plans allowing deap to run on gpu

### ISSUE THREAD

**Issue by kirk86** [DEAP/deap#209]
Hi, may I ask if there are any plans on allowing deap to harness gpu power? With all those gpu libraries out there it might be worth considering whether we could harvest some of those in order to speed up deap.

**Comment by gfviegas** [comment]
that'd be a great deal, and I wonder if anyone ever done that using DEAP. I'm working in a heavy recursive operator in-between the generation proccess and any extra resource with performance would help a ton

**Comment by yxliang01** [comment]
Wondering, can you simply apply numba? :D

**Comment by fmder** [comment]
Yeah you should look to apply GPU in you operators, like initialization,
evaluation and variation.

Le sam. 12 oct. 2019 04 h 25, Xiao Liang <notifications@github.com> a
écrit :
